Content.Shared/Strip/Components/ThievingComponent.cs

```diff
@@ -1,22 +1,49 @@
+using Content.Shared.Actions;
+using Robust.Shared.Prototypes;
+
 namespace Content.Shared.Strip.Components;
 
 /// <summary>
 /// Give this to an entity when you want to decrease stripping times
 /// </summary>
 [RegisterComponent]
-public sealed partial class ThievingComponent : Component
+[Virtual]
+public partial class ThievingComponent : Component
```

Comment:
do not do this dear christ

Reply:
Oh dear god. Yeah Definately do not do this, component inheritance is NOT supported and was deprecated a looong time ago.
If you want to have similar functionality to this, use multiple components and access both of them from a system method

Reply:
The functionality i want - is not to duplicate the fields on the component 3 times.
Eh i can probably work around that

Reply:
both of the other components have nothing to do with thieving and dont need to copy paste their fields anyway
see other reviews